templates/Makefile

```diff
@@ -1,3 +1,8 @@
+init:
+	go mod init <% index .Files `Repository` %>
```

Comment:
since we should know their repo, should we do this for them?

Reply:
Sure, should be fine.